README.md

```diff
@@ -40,10 +40,23 @@ If you are interested in contributing to EvalAI-CLI, follow our [contribution gu
     pip install -e .
     ```
 
-5. Login to cli using the command ``` evalai login```
+5. Configure your Auth Token
+    1. Create an account on the production site (https://evalai.cloudcv.org/)
+    2. Verify your email
+    3. Go to your profile (https://evalai.cloudcv.org/web/profile)
+    4. Click on `Get your Auth Token`
+    5. Copy the token
+    6. Set the token using the `evalai` CLI as shown below
```

Comment:
@drepram can you add a new line after this. It looks cluttered in markdown version